invenio_users_resources/records/mappings/os-v1/groups/group-v1.0.0.json

```diff
@@ -28,6 +28,9 @@
           }
         }
       },
+      "description": {
```

Comment:
topic to raise: same old question, this forces people to delete indexes and reindex. Creating a v2 would allow just querying the alias and getting them all (without having to delete/reindex, i.e. avoiding users functionality downtime). This is also possible since description is not a mandatory field.